HPXMLtoOpenStudio/tests/test_defaults.rb

```diff
@@ -1053,20 +1053,20 @@ def test_ptac
     hpxml.cooling_systems[0].cooling_shr = 0.75
     hpxml.cooling_systems[0].cooling_capacity = 12345
     hpxml.cooling_systems[0].cooling_efficiency_eer = 12.5
-    hpxml.heating_systems[0].heating_efficiency_percent = 0.98
+    hpxml.cooling_systems[0].attached_heating_system_fuel = HPXML::FuelTypeElectricity
```

Comment:
I didn't test the attached heating system efficiency defaulting for room ac, considering it is a rare system type, and the attached heating system inputs are optional. It is tested for ptac here. Is it good enough?